### PR TITLE
Mobile menu categories removal

### PR DESCRIPTION
**Pull request by Glignos**
* Removed inspire categories from the mobile menu for the users.

### CODE REVIEW

**Comment by jmartinm** [comment]
@Glignos once it's ready, please put some screenshots in this issue with all the cases.

Mobile Menu with `INSPIRE_FULL_THEME=True`
Mobile Menu with `INSPIRE_FULL_THEME=False`

File: inspirehep/modules/theme/static/scss/base/header.scss [hunk 1]
```diff
@@ -748,22 +748,30 @@ a.external-link::after {
 
                 #tools-dropdown i,
                 #dropdown-links-stats i,
-                li:nth-child(3) a:before,
-                li:nth-child(4) a:before {
+                li:nth-child(1) a:before,//changed number of child from 3
```
Comment (jmartinm) [inline inspirehep/modules/theme/static/scss/base/header.scss]: Why was the change needed? Is it working both with INSPIRE_FULL_THEME True and False?

The comment does not give information.

File: inspirehep/modules/theme/static/scss/base/header.scss [hunk 2]
```diff
@@ -748,22 +748,30 @@ a.external-link::after {
...
                     padding: 15px;
                     background-color: $light-blue;
                     border-radius: 50%;
                     margin-right: 15px;
 
                 }
 
-                li.dropdown + li:nth-child(3) a:before {
-                    content: "\f0ee";
+                /*li.dropdown + li:nth-child(3) a:before {
```
Comment (jmartinm) [inline inspirehep/modules/theme/static/scss/base/header.scss]: Why is it commented out? Does it need to be removed?

**Comment by Glignos** [comment]
@jmartinm ready for review

File: inspirehep/modules/theme/templates/inspirehep_theme/header.html [hunk 3]
```diff
@@ -144,10 +146,10 @@
           </li>
           {% endif %}
           <li {% if request.blueprint == blueprint|e %} class="active" {% endif %}>
-            <a href="{{ url_for('inspirehep_literature_suggest.create') }}">{{- _('Submit') -}}</a>
+            <a id="submit" href="{{ url_for('inspirehep_literature_suggest.create') }}">{{- _('Submit') -}}</a>
```
Comment (jmartinm) [inline inspirehep/modules/theme/templates/inspirehep_theme/header.html]: Change to a less common name `header-submit-link`

File: inspirehep/modules/theme/templates/inspirehep_theme/header.html [hunk 4]
```diff
@@ -144,10 +146,10 @@
...
           </li>
           <li>
-            <a href="https://labs.inspirehep.net/help">{{- _('Help') -}}</a>
+            <a id="help" href="https://labs.inspirehep.net/help">{{- _('Help') -}}</a>
```
Comment (jmartinm) [inline inspirehep/modules/theme/templates/inspirehep_theme/header.html]: Change to a less common name `header-help-link`

**Comment by kaplun** [comment]
Hi @Glignos can you attach corresponding screenshots so that we can check? @jmartinm just left for one week, so that we would be able to review the actual output and merge it.

**Comment by jacquerie** [comment]
You should improve your commit messages. Learn Linus Torvalds' guidelines on this: https://github.com/Subsurface-divelog/subsurface/blob/35e48a490bde96f8b8ae474afb4457fc5046d638/README#L92-L119

In particular: use a sensible module name (we have no `templates` module, but you're working on the `theme` module, so you should use that), wrap your body at ~70 characters, sign your commits.

**Pull request closed without merging**